test/unit/specs/vislib/lib/y_axis.js

```diff
@@ -204,6 +222,70 @@ define(function (require) {
           checkRange();
         });
       });
+
+      describe('validate user defined values', function () {
```

Comment:
These all seem to test `_validateUserExtents`, maybe put the name of the function in the describe?